From 038cd02de8db90615cbf3205d8e9f279009c0817 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Mon, 25 Mar 2019 11:20:39 +0800
Subject: [PATCH] arm64: dts: rockchip: fix include search path for
 rk3326-863-cif-sensor.dtsi

This is because of the commit form upstream: commit 5ffa2aed389c
("of: remove arch/$(SRCARCH)/boot/dts from include search path for CPP")

Change-Id: I6dc9eca4357f3425eb18fdc97b53e9923a421c33
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-863-cif-sensor.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-cif-sensor.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-863-cif-sensor.dtsi
index b6f18acefdab..c01f4d014a1b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-cif-sensor.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-cif-sensor.dtsi
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: (GPL-2.0+ OR MIT)
  */
 
-#include "../../../../drivers/soc/rockchip/rk_camera_sensor_info.h"
+#include "../../../../../drivers/soc/rockchip/rk_camera_sensor_info.h"
 /{
 	cif_sensor: cif_sensor {
 		compatible = "rockchip,sensor";
-- 
2.35.3

